src/pages/gen2/build-a-backend/data/data-modeling/relationships/index.mdx

```diff
@@ -220,78 +284,109 @@ You can make a "has one" or "has many" relationship bidirectional with the `belo
 
```

Comment:
such picky, very nit: Since you introduce `hasMany` then `hasOne` above, maybe reverse the order of the bidirectional examples to match?